File: java/logic_1/nearTen.java
package java.logic_1;

/*
Given a non-negative number "num", return true if num is within 2 of a multiple of 10.
Note: (a % b) is the remainder of dividing a by b, so (7 % 5) is 2.

nearTen(12) → true
nearTen(17) → false
nearTen(19) → true
*/

public class nearTen {
    public boolean nearten(int num) {
        return num % 10 <= 2 || num % 10 >= 8;
    }
}

File: python/logic-1/near_ten.py
"""
Given a non-negative number "num", return True if num is within 2 of a multiple of 10.  

near_ten(12) → True
near_ten(17) → False
near_ten(19) → True
"""


def near_ten(num):
    return num % 10 <= 2 or num % 10 >= 8
